app/routes.py

```diff
@@ -7,12 +7,14 @@
 @app.route('/')
 @app.route('/index')
 def index():
+    pageSlug = 'Home'
     home = Home()
     homeData = home.index()
-    return render_template('home/index.html', title='Coding Complexity - Home Page', homeData=homeData)
+    return render_template('home/index.html', title='Coding Complexity - Home Page', homeData=homeData, pageSlug=pageSlug)
 
 @app.route('/product/<int:productId>')
 def product(productId):
+    pageSlug = 'Products'
```

Comment:
What is the intended functionality here?

My expected result is that each product page, instead of redirecting to `http://www.codingcomplexity.com/products/<int:productID>` should instead go to a URL like `http://www.codingcomplexity.com/products/python-fundamentals-course`

For that to work, I would think the `products.yaml` file needs another variable like `product_url_slug` for each product, and that is what would be used for the url.

Reply:
That is not permanent, I will replace that with dynamic URLs using a YAML file. Don't worry about that